other/triplet_sum.py

```diff
@@ -0,0 +1,87 @@
+"""
+Given an array of integers and another integer target,
+we are required to find a triplet from the array such that it's sum is equal to
+the target.
+"""
+from itertools import permutations
+from random import randint
+from timeit import repeat
+from typing import List, Tuple
+
+
+def make_dataset() -> Tuple[List[int], int]:
+    arr = [randint(-1000, 1000) for i in range(10)]
+    r = randint(-5000, 5000)
+    return (arr, r)
+
+
+def triplet_sum1(arr: List[int], target: int) -> Tuple[int, int, int]:

```

Comment:
```suggestion
dataset = make_dataset()


def triplet_sum1(arr: List[int], target: int) -> Tuple[int, int, int]:
```

Reply:
When I do this I get en error, 
```Python
NameError: name 'dataset' is not defined
```

Reply:
Commit this suggestion and you will not get that NameError